README.md

```diff
@@ -1,2 +1,30 @@
-# todo-react-app
-Practice react hooks and react router
```

Comment:
Good work so far. but i would advice you add table of content in your readme

Reply:
Will add this. Thanks